development/tools/jobs/kyma/etcd_tls_setup_job_test.go

```diff
@@ -1,4 +1,4 @@
-package tools_test
+package kyma
```

Comment:
It should be `kyma_test` like rest of the files

Reply:
done